src/composables/feed/useGetAll.ts

```diff
@@ -1,23 +1,29 @@
+import HttpMethod from '@/enum/http-method.enum';
 import INewsFeed from '@/interface/news-feed.interface'
 import client from '@/services/client.service'
-import { ref } from 'vue'
+import { Ref, ref } from 'vue'
 
- function useGetAll() {
+function useGetAll(): {
+  posts: Ref<INewsFeed[]>;
+  getAllFeed: () => Promise<boolean>;
```

Comment:
is there a need to expose this function?

Reply:
no need to expose function since the function is called inside the composable. removed the function in the return.